packages/ui/Changeable/Changeable.js

```diff
@@ -127,35 +128,28 @@ const Changeable = hoc(defaultConfig, (config, Wrapped) => {
 
 		constructor (props) {
 			super(props);
-			let value = props[defaultPropKey];
-			let controlled = false;
-
-			if (prop in props) {
-				if (props[prop] != null) {
-					value = props[prop];
-				}
-
-				controlled = true;
-			}
 
 			this.state = {
-				controlled,
-				value
+				value: getValue(props),
```

Comment:
Isn't `getDerivedStateFromProps` called during mounting?  Do we need to set `value` here?

Reply:
Hmm, yes, that makes a lot of sense!